tests/rados/test_pg_split.py

```diff
@@ -0,0 +1,94 @@
+import time
+import traceback
+
+from ceph.ceph_admin import CephAdmin
+from ceph.rados.core_workflows import RadosOrchestrator
+from ceph.rados.pool_workflows import PoolFunctions
+from tests.rados.rados_test_util import create_pools, write_to_pools
+from tests.rados.stretch_cluster import wait_for_clean_pg_sets
+from utility.log import Log
+from utility.utils import method_should_succeed, should_not_be_empty
+
+log = Log(__name__)
+
+
+def run(ceph_cluster, **kw):
+    """
+    Automates PG split and merge test scenarios.
+    1. Create replicated pool
+    2. Verify the pg count and bulk flag
+    3. Set bulk flag to true
+    4. Wait for pg to be active+clean
+    5. Verify new pg_num has increased and greater that initial pg_num
+    6. Set bulk flag to false
+    7. Wait for pg to be active+clean
+    8. Verify latest pg_num has decreased.
+    """
+    try:
+        log.info(run.__doc__)
+        config = kw["config"]
+        cephadm = CephAdmin(cluster=ceph_cluster, **config)
+        rados_obj = RadosOrchestrator(node=cephadm)
+        pool_obj = PoolFunctions(node=cephadm)
+        client_node = ceph_cluster.get_nodes(role="client")[0]
+        timeout = config.get("timeout", 10800)
+
+        log.info("Running PG split merge scenarios")
+        pool = create_pools(config, rados_obj, client_node)
+        should_not_be_empty(pool, "Failed to retrieve pool details")
+        write_to_pools(config, rados_obj, client_node)
+        rados_obj.change_recover_threads(config=pool, action="set")
```

Comment:
Please add a task to change the code here in ` change_recover_threads() ` to change implementation for 6.x, when you add this test in quincy suite.

Reply:
sure will do that.